### PR TITLE
Support GetTokenSupply and GetProtocolParameters APIs in the Java bin…

### LINKED ISSUE #1306: Support GetTokenSupply and GetProtocolParameters APIs in the Java binding
## Description

Support GetTokenSupply and GetProtocolParameters APIs in the Java binding.

## Reference
https://github.com/iotaledger/iota.rs/pull/1303

## Are you planning to do it yourself in a pull request?

Yes.

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/iota.rs/issues/1306

### CODE REVIEW

File: client/bindings/java/iota-client-java/src/main/java/org/iota/apis/MiscellaneousApi.java [hunk 1]
```diff
@@ -184,5 +185,14 @@ public Map.Entry<BlockId, Block> postBlockPayload(BlockPayload payload) throws C
...
     }
 
+    /**
+     * Returns the protocol parameters.
+     */
+    public ProtocolParametersResponse getProtocolParameters() throws ClientException {
+        JsonObject responsePayload = (JsonObject) callBaseApi(new ClientCommand("getProtocolParameters"));
+        return new ProtocolParametersResponse(responsePayload);
+    }
+
```
Comment (Thoralf-M) [inline client/bindings/java/iota-client-java/src/main/java/org/iota/apis/MiscellaneousApi.java:195]:
```suggestion
```

File: client/bindings/java/iota-client-java/CHANGELOG.md [hunk 2]
```diff
@@ -1,29 +1,13 @@
...
-### Changed
-
-### Deprecated
-
-### Removed
-
-### Fixed
-
-### Security -->
-
```
Comment (Thoralf-M) [inline client/bindings/java/iota-client-java/CHANGELOG.md:21]: This part should stay?

Comment (thibault-martinez) [reply client/bindings/java/iota-client-java/CHANGELOG.md:21]: Yes

Comment (samuel-rufi) [reply client/bindings/java/iota-client-java/CHANGELOG.md:21]: Added

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**